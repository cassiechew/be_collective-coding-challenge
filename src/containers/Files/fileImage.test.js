import { create } from 'react-test-renderer';
import React from 'react';
import FileImage from './fileImage';

describe('snapshot test', () => {
    test('testing footer', () => {
      let tree = create(<FileImage src="source/url" />);
      expect(tree.toJSON()).toMatchSnapshot();
    });
});