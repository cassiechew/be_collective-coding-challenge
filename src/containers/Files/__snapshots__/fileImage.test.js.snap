// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test testing footer 1`] = `
<div>
  <img
    alt="failed"
    src="source/url"
  />
   
</div>
`;
